src/test/java/com/plaid/client/integration/AbstractItemIntegrationTest.java

```diff
@@ -43,22 +49,31 @@ protected static void assertAccount(Account actualAccount,
 
   @Before
   public void setUpItem() throws Exception {
-    Response<ItemCreateResponse> response =
-      client().service()
-        .itemCreate(new ItemCreateRequest(
-          setupItemInstitutionId(),
-          setupItemProducts())
-          .withCredentials("username", "user_good")
-          .withCredentials("password", "pass_good")).execute();
+    Response<SandboxPublicTokenCreateResponse> createResponse =
+      client().service().sandboxPublicTokenCreate(new SandboxPublicTokenCreateRequest(setupItemInstitutionId(), setupItemProducts())).execute();
+    assertSuccessResponse(createResponse);
+    Response<ItemPublicTokenExchangeResponse> response =
+      client().service().itemPublicTokenExchange(new ItemPublicTokenExchangeRequest(createResponse.body().getPublicToken())).execute();
     assertSuccessResponse(response);
-    itemCreateResponse = response.body();
+
+    exchangeTokenResponse = response.body();
```

Comment:
can we add this. in front of setting these fields

Reply:
:zap: